server/statshouse/statshouse-metrics.h

```diff
@@ -0,0 +1,87 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2023 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <cassert>
+
+#include "common/dl-utils-lite.h"
+#include "common/mixin/not_copyable.h"
+#include "runtime/memory_resource/memory_resource.h"
+#include "server/job-workers/job-stats.h"
+#include "server/statshouse/statshouse-client.h"
+#include "server/workers-control.h"
+#include "server/workers-stats.h"
+
+enum class script_error_t : uint8_t;
+
+class StatsHouseMetrics : vk::not_copyable {
```

Comment:
What about renaming this to StatshouseManager or something else?

Reply:
done